Project.toml

```diff
@@ -31,7 +31,7 @@ SuiteSparse = "4607b0f0-06f3-5cda-b6b1-a6196a1729e9"
 [compat]
 AmplNLReader = "~0.9"
 BinaryProvider = "~0.5"
-CUDA = "~1.3, ~2, 3.0"
+CUDA = "~3.3"
```

Comment:
I am wondering if that bound is too restrictive? Do we need `CUDA.jl` v3.3 to use CUDA 11.3.1?

Reply:
I am wondering whether, instead of forcing the CUDA version to be `>= 3.3`, we could instead add a `if` statement in the code to load the function `sytrs` only if `CUDA 11.3` is detected on the system. By doing that, we would avoid a breaking change.

Reply:
That might be a better idea 😊